Add custom functions to create close cookies

Abban opened the pull request:
Calling `mw.centralNotice.hideBanner()` causes a spike in false tracking events during the fundraising campaign, due to the function looping the multiple URLS contained in `wgNoticeHideUrls` in order to set cookies on multiple wikis.

This replaces the `hideBanner` call with custom setters that only create the cookies on wikipedia.org.

See https://phabricator.wikimedia.org/T344381

tzhelyazkova commented:
The CI will keep failing because the `js` refers to html elements that have been removed.

Abban committed:
Add custom functions to create close cookies

Calling `mw.centralNotice.hideBanner()` causes a spike in false
tracking events during the fundraising campaign, due to the
function looping the multiple URLS contained in `wgNoticeHideUrls`
in order to set cookies on multiple wikis.

This replaces the `hideBanner` call with custom setters that only
create the cookies on wikipedia.org.

See https://phabricator.wikimedia.org/T344381
